lib/hsa/mcwamp_hsa.cpp

```diff
@@ -4856,7 +4853,7 @@ bool HSABarrier::isReady() override {
 }
 
 bool HSACopy::isReady() override {
-    bool ready = (hsa_signal_load_acquire(signal) == 0);
+    bool ready = (hsa_signal_load_acquire(_signal) == 0);
```

Comment:
Unless I am mistaken, this is deprecated, with `hsa_signal_load_scacquire` being the current / preferred form.